zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -147,6 +147,10 @@ impl<F: Field> ReversionInfo<F> {
             .assign(region, offset, Value::known(F::from(is_persistent as u64)))?;
         Ok(())
     }
+
+    pub(crate) fn rw_delta(&self) -> Expression<F> {
+        2.expr()
+    }
```

Comment:
I can see that this block also exists in Scroll's `zkevm-circuits` repo. However I believe it will be good to add a descriptive comment for this `2.expr()`